### ISSUE TITLE
After upgrade to NC 18.0.1 user settings are shown mixed up

### ISSUE THREAD

**Issue by m8ichael** [nextcloud/server#19809]
### Steps to reproduce
1. Add several nextcloud users
2. Add users to several groups with long group names

### Expected behaviour
The users should be displayed in an ordered table.

### Actual behaviour
The user table is displayed chaotically.

### Server configuration

**Operating system:**
Ubuntu 16.04

**Web server:**
Apache2

**Database:**

**PHP version:**
7.3

**Nextcloud version:** (see Nextcloud admin page)
18.0.1

**Updated from an older Nextcloud/ownCloud or fresh install:**
Upgraded from 17.0.3

**Where did you install Nextcloud from:**
Updater

**Signing status:**
<details>
<summary>Signing status</summary>

```
No errors have been found.
```
</details>

**List of activated apps:**
<details>
<summary>App list</summary>

```
Enabled:
  - accessibility: 1.4.0
  - activity: 2.11.0
  - apporder: 0.9.0
  - bruteforcesettings: 1.5.0
  - calendar: 2.0.2
  - circles: 0.18.3
  - cloud_federation_api: 1.1.0
  - comments: 1.8.0
  - contacts: 3.2.0
  - dav: 1.14.0
  - federatedfilesharing: 1.8.0
  - federation: 1.8.0
  - files: 1.13.1
  - files_external: 1.9.0
  - files_mindmap: 0.0.17
  - files_rightclick: 0.15.2
  - files_sharing: 1.10.1
  - files_trashbin: 1.8.0
  - files_versions: 1.11.0
  - groupfolders: 6.0.1
  - impersonate: 1.5.0
  - logreader: 2.3.0
  - lookup_server_connector: 1.6.0
  - mail: 1.1.3
  - nextcloud_announcements: 1.7.0
  - notifications: 2.6.0
  - oauth2: 1.6.0
  - ocr: 6.0.23
  - onlyoffice: 4.1.4
  - password_policy: 1.8.0
  - polls: 1.3.0
  - privacy: 1.2.0
  - provisioning_api: 1.8.0
  - recommendations: 0.6.0
  - serverinfo: 1.8.0
  - settings: 1.0.0
  - sharebymail: 1.8.0
  - systemtags: 1.8.0
  - tasks: 0.11.3
  - text: 2.0.0
  - theming: 1.9.0
  - twofactor_backupcodes: 1.7.0
  - twofactor_totp: 4.1.2
  - updatenotification: 1.8.0
  - user_ldap: 1.8.0
  - viewer: 1.2.0
  - workflowengine: 2.0.0

```
</details>

**Nextcloud configuration:**
<details>
<summary>Config report</summary>

```
 "system": {
        "instanceid": "***REMOVED SENSITIVE VALUE***",
        "passwordsalt": "***REMOVED SENSITIVE VALUE***",
        "secret": "***REMOVED SENSITIVE VALUE***",
        "trusted_domains": [
            "cloud.xyz.de"
        ],
        "datadirectory": "***REMOVED SENSITIVE VALUE***",
        "dbtype": "mysql",
        "version": "18.0.1.3",
        "dbname": "***REMOVED SENSITIVE VALUE***",
        "dbhost": "***REMOVED SENSITIVE VALUE***",
        "dbport": "",
        "dbtableprefix": "oc_",
        "dbuser": "***REMOVED SENSITIVE VALUE***",
        "dbpassword": "***REMOVED SENSITIVE VALUE***",
        "default_language": "de",
        "default_locale": "de_DE",
        "installed": true,
        "remember_login_cookie_lifetime": 1296000,
        "session_lifetime": 86400,
        "session_keepalive": true,
        "skeletondirectory": "",
        "lost_password_link": "disabled",
        "ldapIgnoreNamingRules": false,
        "ldapProviderFactory": "\\OCA\\User_LDAP\\LDAPProviderFactory",
        "mail_from_address": "***REMOVED SENSITIVE VALUE***",
        "mail_smtpmode": "smtp",
        "mail_smtpauthtype": "LOGIN",
        "mail_domain": "***REMOVED SENSITIVE VALUE***",
        "defaultapp": "calendar",
        "auth.bruteforce.protection.enabled": true,
        "maintenance": false,
        "updater.release.channel": "stable",
        "theme": "",
        "loglevel": 0,
        "forcessl": true,
        "forceSSLforSubdomains": true,
        "trashbin_retention_obligation": "7, 20",
        "has_internet_connection": true,
        "check_for_working_webdav": true,
        "check_for_working_wellknown_setup": true,
        "check_for_working_htaccess": true,
        "cron_log": true,
        "logtimezone": "Europe\/Berlin",
        "logdateformat": "F d, Y H:i:s",
        "ldapUserCleanupInterval": 51,
        "memcache.local": "\\OC\\Memcache\\Redis",
        "redis": {
            "host": "***REMOVED SENSITIVE VALUE***",
            "port": 6379
        },
        "memcache.locking": "\\OC\\Memcache\\Redis",
        "mail_smtpsecure": "ssl",
        "mail_smtpauth": 1,
        "mail_smtphost": "***REMOVED SENSITIVE VALUE***",
        "mail_smtpport": "465",
        "mail_smtpname": "***REMOVED SENSITIVE VALUE***",
        "mail_smtppassword": "***REMOVED SENSITIVE VALUE***",
        "overwrite.cli.url": "https:\/\/cloud.xyz.de",
        "app_install_overwrite": [
            "calendar"
        ],
        "updater.secret": "***REMOVED SENSITIVE VALUE***"
    }
}

```
</details>

**Are you using external storage, if yes which one:** no

**Are you using encryption:** no

**Are you using an external user-backend, if yes which one:** ActiveDirectory

### Client configuration
**Browser: Firefox 73.0.1**

**Operating system:**
Windows 10

**Comment by kesselb** [comment]
Duplicate of https://github.com/nextcloud/server/issues/18920

**Comment by kesselb** [comment]
Thanks 👍

Will be fixed with 18.0.2.
